peachjam/adapters/adapters.py

```diff
@@ -92,10 +96,23 @@ def get_doc_list(self):
             url = res["next"]
         return results
 
-    def get_updated_documents(self, last_refreshed):
+    def check_for_deleted(self, docs):
+        uris = {d["expression_frbr_uri"] for d in docs}
+        for doc in docs:
+            for pit in doc["points_in_time"]:
+                for expr in pit["expressions"]:
+                    uris.add(expr["expression_frbr_uri"])
+
+        deleted = Legislation.objects.exclude(
```

Comment:
Just remember this must be scoped to only the place (country + locality) that the ingestor is applicable to. eg. AfricanLII we have four separate ingestors (EAC, ECOWAS, SADC, AU). They should only delete "their" legislation.

Reply:
I've added the jurisdiction and locality as settings which we can use here.